gramex/data.py

```diff
@@ -212,6 +214,8 @@ def filter(url, args={}, meta={}, engine=None, ext=None, columns=None,
     })
     controls = _pop_controls(args)
     transform = _transform_fn(transform, transform_kwargs)
+    if type(query) == str and 'url' in args.keys() and args['url'].startswith('mongodb'):
```

Comment:
We don't want anything mongodb (or any DB) specific in filter(). That should be part of the plugins only.

Reply:
BTW, we don't want to eval it. Just take the object as-is and make it work.